### PR TITLE
God script

### PR DESCRIPTION
**Pull request by NSmeef**

### CODE REVIEW

**Comment by NSmeef** [comment]
So the `publishAll` script is not properly robust due to having a `set -ex` flag in there which is `verbose` and `fail on non-zero exit code`. So if rm -rf on one of the directories has nothing in it, it will fail.

Will remove and just get verbose mode in.

**Comment by NSmeef** [comment]
cant use default-jdk as stardog binaries require a later version of java.

File: tools/publishAll [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -ex
+
+deletedb "$DBNAME"
+
+#Delete git stuffs
+find . | grep .git | xargs rm -rf
```
Comment (jameskirk84) [inline tools/publishAll]: can this just be rm -rf .git .gitignore ?

**Comment by jameskirk84** [comment]
looks good - the only thing to change is the removing the .git and .gitignore directories in a cleaner fashion

**Pull request merged**